sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/ecert-integration/_tests_/ecert-part-time-process-integration.scheduler.e2e-spec.ts

```diff
@@ -58,6 +58,13 @@ describe(
       processor = app.get(PartTimeECertProcessIntegrationScheduler);
       systemUsersService = app.get(SystemUsersService);
       gcNotifyService = app.get(GCNotifyService);
+      // Insert fake email contact to send ministry email.
+      await db.notificationMessage.update(
+        {
+          id: NotificationMessageType.MinistryNotificationDisbursementBlocked,
+        },
+        { emailContacts: ["shashank.x.shekhar@gov.bc.ca"] },
```

Comment:
Are you sure you're going to use your real email?

Reply:
I had to use one of the emails present in the GC Notify emails to send list. It won't let me send an email to an email address not present in the list.

Reply:
E2E tests are not supposed to send emails to GC Notify. Can you please elaborate on why are we doing this?

Reply:
Oops, my bad. Missed the part that here we just check if an entry has been made to the `notifications` table. Updated it to a dummy email. Thanks.